pkg/volume/cephfs/cephfs_test.go

```diff
@@ -127,7 +127,7 @@ func TestConstructVolumeSpec(t *testing.T) {
 		t.Errorf("can't find cephfs plugin by name")
 	}
 
-	cephfsSpec, err := plug.(*cephfsPlugin).ConstructVolumeSpec("cephfsVolume", "/cephfsVolume/")
+	cephfsSpec, _ := plug.(*cephfsPlugin).ConstructVolumeSpec("cephfsVolume", "/cephfsVolume/")
```

Comment:
should we check `err`, just to be sure?

Reply:
yes, it is better to check the `err` , let me update it.